### PR TITLE
Add error_code column to the envelope_events table

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1096

### Change description ###
Add error_code  to envelope_events table.
This will be used to send the error notification message.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/blobrouter/services/EnvelopeService.java [hunk 1]
```diff
@@ -92,7 +93,12 @@ public void markAsRejected(UUID id, String reason) {
                 env -> {
                     envelopeRepository.updateStatus(id, Status.REJECTED);
                     envelopeRepository.updatePendingNotification(id, true); // notification pending
-                    eventRepository.insert(new NewEnvelopeEvent(id, EventType.REJECTED, reason));
+                    eventRepository.insert(new NewEnvelopeEvent(
+                        id,
+                        EventType.REJECTED,
+                        ErrorCode.ERR_AV_FAILED,
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/services/EnvelopeService.java:99]: why this code?

Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/EnvelopeService.java:99]: The error code is needed for the notification message to let Exela know why it is failed. Notification Service which consumes the messages from notification is expecting the `error_code` with ErrorCode (enum)

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/EnvelopeService.java:99]: I mean, why `ERR_AV_FAILED`?

Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/EnvelopeService.java:99]: it can be any error type, doesn't have to be `ERR_AV_FAILED`

Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/EnvelopeService.java:99]: oh sorry, I didn't notice that. thought it's a test. will update

Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/EnvelopeService.java]: This is causing several changes. For now, I've set the value as null. 
In the next PR, I'll add error_code parameter to the method, and update the references.

**Pull request merged**